src/org/labkey/test/components/ui/domainproperties/samples/SampleTypeDesigner.java

```diff
@@ -62,87 +62,6 @@ public T addParentAlias(String alias, @Nullable String optionDisplayText)
         return getThis();
     }
 
```

Comment:
Removing this code left some unused imports :
```
import org.labkey.test.components.html.Input;
import org.labkey.test.components.react.ReactSelect;
```
May want to clean those up.

Reply:
Thanks. Done.